src/components/NavigationBar/index.js

```diff
@@ -327,6 +327,10 @@ class NavigationBar extends Component {
 
   handleLanguageChange = (event, index, values) => {
     localStorage.setItem('languages', event.target.value);
+    let languages = [...event.target.value];
+    if (languages.length === 0) {
+      event.target.value.push('en');
```

Comment:
Do not push something inside the event. Instead, push into the `languages` variable.